akka-typed/src/main/scala/akka/typed/receptionist/Receptionist.scala

```diff
@@ -0,0 +1,103 @@
+package akka.typed.receptionist
+
+import akka.typed.ActorRef
+import akka.typed.ActorSystem
+import akka.typed.Extension
+import akka.typed.ExtensionId
+import akka.typed.internal.receptionist.LocalReceptionist
+
+import scala.concurrent.duration._
+import scala.reflect.ClassTag
+
+class ReceptionistExt(system: ActorSystem[_]) extends Extension {
+  val receptionist: ActorRef[Receptionist.Command] =
+    ActorRef(
+      system.systemActorOf(
+        LocalReceptionist.behavior // FIXME: either create local or cluster version depending on configuration
+        , "receptionist")(
+        // FIXME: where should that timeout be configured? Shouldn't there be a better `Extension`
+        //        implementation that does this dance for us?
```

Comment:
You mean the awaiting to hand out a proper ref? Sounds like a good candidate to provide a helper for